### ISSUE TITLE
Support for Cordova 4.0

### ISSUE THREAD

**Issue by jvence** [dankurka/gwtphonegap#64]
Will this be upgraded to support the new 4.0?

**Comment by Jul13nT** [comment]
I would like to know as well when it will be upgraded to support a more recent version. I need at least the 4.1.1 version as Google said : https://support.google.com/faqs/answer/6325474

**Comment by svkirans** [comment]
Yes, even I have the same issue. Some of my apps are facing that issue. I
started working on the upgrade but got pulled into other things. I think I
should be able to complete it in 2 weeks.

On Fri, Feb 19, 2016 at 6:08 PM, Julien Torrielli notifications@github.com
wrote:

> I would like to know as well when it will be upgraded to support a more
> recent version. I need at least the 4.1.1 version as Google said : [
> https://support.google.com/faqs/answer/6325474]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dankurka/gwtphonegap/issues/64#issuecomment-186199736
> .

## 

Thanks & Regards,
-Kiran
